src/basic/string-util.c

```diff
@@ -145,57 +145,33 @@ char *strnappend(const char *s, const char *suffix, size_t b) {
 
 char *strjoin_real(const char *x, ...) {
         va_list ap;
-        size_t l;
+        size_t l = 0;
         char *r, *p;
+        const char *t;
```

Comment:
Nowadays we define iterator variables directly in the loop (when they are not used after the loop)...